gen/container_test.go

```diff
@@ -1319,12 +1321,12 @@ func TestContainerValidate(t *testing.T) {
 					"foo":   nil,
 				},
 			},
-			wantError: "invalid [foo]: value is nil",
+			wantError: "invalid map 'map[string][]byte', key [foo]: value is nil",
 		},
 	}
 
 	for _, tt := range tests {
-		t.Run(tt.desc, func(t *testing.T) {
+		t.Run(tt.desc+" wire", func(t *testing.T) {
```

Comment:
optional nit: can use `/wire` here and `/streaming` below to have the test system treat them as subtests.

```suggestion
		t.Run(tt.desc+"/wire", func(t *testing.T) {
```